sdk/cosmosdb/cosmos/samples/v3/typescript/src/SasTokenAuth.ts

```diff
@@ -9,11 +9,11 @@ import path from "path";
 import * as dotenv from "dotenv";
 dotenv.config({ path: path.resolve(__dirname, "../sample.env") });
 
-import { SasTokenProperties } from "../dist-esm/client/SasToken/SasTokenProperties";
-import { SasTokenPermissionKind } from "../dist-esm/common/constants";
-import { createAuthorizationSasToken } from "../dist-esm/utils/SasToken";
+import { SasTokenProperties } from "../../../../dist-esm/client/SasToken/SasTokenProperties";
+import { SasTokenPermissionKind } from "../../../../dist-esm/common/constants";
+import { createAuthorizationSasToken } from "../../../../dist-esm/utils/SasToken";
```

Comment:
All imports from dist-esm need to be rewritten to import from `@azure/cosmos`.

Reply:
Thank you. It's my fault